pt-BR/mail/chrome/messenger/preferences/display.dtd

```diff
@@ -29,64 +29,64 @@
 <!ENTITY removeTagButton.accesskey "x">
 
 <!-- Fonts -->
-<!ENTITY fontsAndColors.label    "Fonts">
-<!ENTITY defaultFont.label       "Default font:">
+<!ENTITY fontsAndColors.label    "Fontes">
+<!ENTITY defaultFont.label       "Fonte padrão:">
 <!ENTITY defaultFont.accesskey   "D">
-<!ENTITY defaultSize.label       "Size:">
+<!ENTITY defaultSize.label       "Tamanho:">
 <!ENTITY defaultSize.accesskey   "S">
 <!ENTITY fontOptions.accesskey   "A">
 <!ENTITY fontOptions.label       "Avançado…">
 
-<!ENTITY pb.messagelist.colortagrow.label "Apply the topic color to the entire row in the message list pane">
+<!ENTITY pb.messagelist.colortagrow.label "Aplique a cor na linha inteira no painel de lista de mensagens">
 <!ENTITY pb.messagelist.colortagrow.accesskey "g">
 
 <!-- conversation preferences -->
-<!ENTITY pb.itemConversation.label        "Conversations">
-<!ENTITY pb.conversations.display.label "Include messages from:">
-<!ENTITY pb.conversations.folderonly.label1 "The current folder">
-<!ENTITY pb.conversations.all.label1 "Every folder">
+<!ENTITY pb.itemConversation.label        "Conversas">
+<!ENTITY pb.conversations.display.label "Incluir mensagens de:">
+<!ENTITY pb.conversations.folderonly.label1 "Pasta atual">
+<!ENTITY pb.conversations.all.label1 "Todas as pastas">
 
-<!ENTITY pb.conversations.sortorder.label "Order conversations by:">
-<!ENTITY pb.conversations.sortoldest.label "Oldest messages first">
-<!ENTITY pb.conversations.sortnewest.label "Newest messages first">
+<!ENTITY pb.conversations.sortorder.label "Ordenar conversas pelas:">
+<!ENTITY pb.conversations.sortoldest.label "Mensagens mais antigas ptimeiro">
```

Comment:
I decided to keep "Mensagens" in the beginning because of "pelas" right before. Please let me know if you agree with this. I've fixed ptimeiro though.